packages/express/test/unit/express_mw.test.js

```diff
@@ -5,8 +5,8 @@ var sinon = require('sinon');
 var sinonChai = require('sinon-chai');
 
 var expressMW = require('../../lib/express_mw');
-var SegmentEmitter = require('../../../core/lib/segment_emitter.js');
-var ServiceConnector = require('../../../core/lib/middleware/sampling/service_connector.js');
+var SegmentEmitter = require('../../../core/dist/lib/segment_emitter.js');
```

Comment:
Is the `dist` repo going to be existed in the final assets that are packaged?

Reply:
Yep, the change in 3.3.2 was to repackage all of the content from `lib` into a `dist` directory, which is what's ultimately tested against and vended to customers.

Reply:
These unit tests were passing before because when you clone the source repo and build the SDK, you have access to BOTH the `dist` and `lib` directories. But once it is vended to customers, only the `dist` is available.

Reply:
Thanks for the clarification as I was thinking why unit tests didn't catch this